outlook/src/classes/EnrichmentInfo.ts

```diff
@@ -30,6 +31,8 @@ class EnrichmentInfo {
                 return "";
             case EnrichmentInfoType.CompanyCreated:
                 return _t("Company created");
+            case EnrichmentInfoType.CompanyUpdated:
+                return _t("Company updated");
```

Comment:
Is this sentence already translated ?

Reply:
Since this is targeting saas-14.3 and translations exist in 14.4 I believe, new translations will only be added in the forward port.

Reply:
Okay, PR targeting 14.4 added.

Here is the link to the PR:
https://github.com/odoo/odoo/pull/73622